packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/VariableStarViewer/VariableStarViewer.js

```diff
@@ -245,7 +250,7 @@ VariableStarViewer.propTypes = {
     chartOptions: PropTypes.object
   }),
   rawJSON: PropTypes.shape({
-    data: PropTypes.array,
+    data: PropTypes.oneOfType([ PropTypes.array, PropTypes.object ]),
```

Comment:
Can this still be an array? Looks like it is only an object here.

Reply:
Actually, no it shouldn't be an array. Thanks.